hugo/assets/scss/components/tag.scss

```diff
@@ -31,6 +36,11 @@
         width: 12px;
     }
 
+    &--large {
+        min-height: 44px;
+        padding: 0 21px 0 18px;
```

Comment:
is there a reason we use pixels here instead of rem?

Reply:
no, but I will use rem from now on!